charts/fluent-bit/values.yaml

```diff
@@ -29,7 +29,16 @@ podSecurityPolicy:
 podSecurityContext:
   {}
   # fsGroup: 2000
-
+podDnsConfig: {}
```

Comment:
```suggestion
dnsConfig: {}
```

nit: no need to prefix with `pod`, reasoning: just above `podSecurityContext` has the pod prefix as it is opposed to the container level security context, whereas dnsConfig is always at the pod level